css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,51 @@
+$breadcrumb-font-size: $document-font-size !default;
+$breadcrumb-spacing-horizontal-start: 0.5em !default;
+$breadcrumb-spacing-horizontal-end: 0.25em !default;
+$breadcrumb-padding-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
```

Comment:
I know we're doing this already, so it has passed several accessibility reviews, just curious how this reads and whether it should be using a rotated border instead. Feel free to resolve.

Reply:
It reads "Slash".